doc/developer/webservices.rst

```diff
@@ -58,7 +58,7 @@ OGC Server
         "wfsSupport": (true|false),
         "imageType": "image/(jpeg|png)",
         "isSingleTile": (true|false),
-        "type": "(mapserver|geoserver|qgisserver)",
+        "type": "(mapserver|geoserver|qgisserver|arcgis)",
```

Comment:
no other here ?

Reply:
Effectively other is missing :-), thanks